pkg/lib/dialogs.tsx

```diff
@@ -82,99 +82,92 @@
  *   }
  * }
  *
- * If there is a situation where you want to wait until a Dialog is closed
- * after opening, you can "await Dialogs.show(<My Dialog />)"
+*
+ * - Dialogs.show(component)
  *
- * class Example extends React.Component {
- *   static contextType = DialogsContext;
+ * Calling "Dialogs.show" will render the given component as a direct
+ * child of the inner-most enclosing "WithDialogs" component.  The
+ * component is of course intended to be a dialog, such as
+ * Patternfly's "Modal".  There is only ever one of these; a second
+ * call to "show" is considered a bug and "Dialogs.close" should be called first.
```

Comment:
See above, s/should/must/

Reply:
This is just text moved from one part of the file to another, but I'll fix it.